FIAT/raviart_thomas.py

```diff
@@ -20,7 +20,7 @@ def RTSpace(ref_el, degree):
     sd = ref_el.get_spatial_dimension()
 
     k = degree - 1
-    vec_Pkp1 = polynomial_set.ONPolynomialSet(ref_el, k + 1, (sd,))
```

Comment:
Also here. Do we really need to change this file?

Reply:
as above